src/current/v24.3/troubleshoot-lock-contention.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Troubleshoot Lock Contention
+summary: This tutorial presents how to understand lock contention, how to identify waiting and blocking transactions, and possible ways to remediate lock contention.
+toc: true
+---
+
+Lock contention is a type of [transaction contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) that occurs when a [transaction]({% link {{ page.version.version }}/transactions.md %}) is unable to complete due to another concurrent or recent transaction attempting to write to the same data. Lock contention may be the cause of [slow SQL query performance]({% link {{ page.version.version }}/eventlog.md %}#sql-slow-query-log).
+
+This tutorial presents:
+
+- How to [understand lock contention](#understand-lock-contention) by reproducing a basic example.
+- How to [identify waiting and blocking transactions](#identify-waiting-and-blocking-transactions) by using the Insights page of the DB Console.
+- Possible ways to [remediate lock contention](#remediate-lock-contention).
+
+## Before you begin
+
+[Download]({% link releases/index.md %}) and [Install CockroachDB]({% link {{ page.version.version }}/install-cockroachdb-mac.md %}).
+
+### Terminal 1
+
+The examples in this tutorial will use three terminals, one for each transaction.
```

Comment:
It seems like this line "The examples in this tutorial..." could precede the "Terminal 1" heading. Since it is really about all of the following sections.